typescript-rest/src/cursor.ts

```diff
@@ -14,10 +14,11 @@ export class TDengineCursor {
         this._uri = {
             host: options.host,
             path: options.path,
+            cloudUri: options.cloudUri,
             port: options.port,
-            schema:options.schema,
-            query:options.query,
-            fragment:options.fragment,
+            schema: options.schema,
+            query: options.query,
+            fragment: options.fragment, 
```

Comment:
extra space

Reply:
this space is added atuomatically after using VS code "Format Document" option.